content/docs/command-reference/remote/modify.md

```diff
@@ -236,8 +236,8 @@ these settings, you could use the following options.
 To communicate with a remote object storage that supports an S3 compatible API
 (e.g. [Minio](https://min.io/),
 [DigitalOcean Spaces](https://www.digitalocean.com/products/spaces/),
-[IBM Cloud Object Storage](https://www.ibm.com/cloud/object-storage) etc.) you
-must explicitly configure the `endpointurl`:
```

Comment:
not sure about this one ... you must do this indeed

Reply:
Yes, that's what this says now (it's just a long sentence): "To communicate with ... an S3 compatible API ... configure the remote's `endpointurl` explicitly". The word must is very emphatic, I don't think we needed it here. It's still imperative sentence, so it tells you what you have to do, basically.